src/store/getters/state-getters.ts

```diff
@@ -91,13 +91,13 @@ export const isRestorationFiling = (state: StateIF): boolean => {
 }
 
 /** Whether the current filing is a Limited Extension Restoration. */
-export const isLimitedExtendRestorationFiling = (state: StateIF): boolean => {
-  return (getRestoration(state).type === RestorationTypes.LTD_EXTEND)
+export const isLimitedExtendRestorationFiling = (state: StateIF, getters): boolean => {
+  return (getters.getRestoration.type === RestorationTypes.LTD_EXTEND)
```

Comment:
Converting to pure functions as suggested by Vuex documentation.

Reply:
I'm pleased to have a suggestion on this, but what is the type of `getters.getRestoration`?

Reply:
That's a good question!  If it's okay with you, I'd like to change the way we populate the store with the restoration filing.  At present, the `parseRestorationFiling()` method, set individual values with methods like: `setRestorationType()` and `setRestorationExpiry()`.  What I'd prefer to do, is load the following three resources into the store and use getters to extract the data we need.  

- restoration filing
- entity snapshot
- state filing

This approach greatly reduces the complexity of populating the store and it gives us more flexibility to get whatever data we need -- both now and in the future.

Reply:
Sure, go for it. (Don't get bogged down.)

Reply:
What do you think about creating a separate ticket for the work so it's visible and gets approval before proceeding?   Is that worthwhile? There may be other refactoring tasks that are higher priority.

Reply:
I think we wouldn't prioritize a refactoring ticket right now, but if I found some easy changes that moved towards our vision, I'd probably do them as part of existing work.